.github/workflows/release.yaml

```diff
@@ -210,4 +230,4 @@ jobs:
         if: github.ref == 'refs/heads/release'
         working-directory: ./editors/code
         # token from https://dev.azure.com/rust-analyzer/
-        run: npx vsce publish --pat ${{ secrets.MARKETPLACE_TOKEN }} --packagePath ../../dist/rust-analyzer.vsix
+        run: npx vsce publish --pat ${{ secrets.MARKETPLACE_TOKEN }} --packagePath ../../dist/rust-analyzer-*.vsix
```

Comment:
will this do the right thing? I am thinking about `--packagePath foo bar` vs `--packagePath foo --packagePath bar`.

Reply:
I home so, the help says:

      -i, --packagePath <paths...>    Publish the provided VSIX packages.